packages/integration/lib/src/views/media/media_preview_screen.dart

```diff
@@ -60,10 +63,9 @@ class _LMFeedMediaPreviewScreenState extends State<LMFeedMediaPreviewScreen> {
 
   @override
   Widget build(BuildContext context) {
-    final DateFormat formatter = DateFormat('MMMM d, hh:mm');
     final String formatted = formatter.format(post.createdAt);
-    final LMFeedThemeData feedTheme = LMFeedTheme.of(context);
-    return Scaffold(
+    return widgetUtility.scaffold(
```

Comment:
No builder classes support in this screen?

Reply:
This screen do not have any customisability as of now, we can align it in the coming release